Cargo.toml

```diff
@@ -7,7 +7,7 @@ documentation = "https://docs.rs/structopt"
 repository = "https://github.com/TeXitoi/structopt"
 keywords = ["clap", "cli", "derive", "docopt"]
 categories = ["command-line-interface"]
-license = "WTFPL"
+license = "Apache-2.0/MIT"
```

Comment:
Please change that to `MIT OR Apache-2.0` as in https://github.com/rust-lang/cargo/blob/master/Cargo.toml#L7

Reply:
@TeXitoi Actually `Apache-2.0/MIT` is the correct syntax. See https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata.

```toml
# This is a string description of the license for this package. Currently
# crates.io will validate the license provided against a whitelist of known
# license identifiers from http://spdx.org/licenses/. Multiple licenses can be
# separated with a `/`.
license = "..."
```

Reply:
OK. My bad. Strange that cargo doesn't validate its own format.